src/main/resources/default/assets/scripts/blob-upload-url-popover.js

```diff
@@ -12,16 +12,19 @@
 function initUrlUploadPopover(_triggerElement, _uploadContainer, _resetButton, _fileNameContainer, i18n, updateCallback) {
```

Comment:
I could not test this code, no `<t:blobImageSoftRefField>` in OX

Reply:
I´ll test it  in S2 and adjust it if needed